includes/class-admin.php

```diff
@@ -695,4 +689,35 @@ private function upgrade_to_1_0_0() {
 		// Update the option.
 		update_option( 'optimizely_post_types', $post_types );
 	}
+
+	/**
+	 * Upgrades the database version of the plugin to version 1.2.0.
+	 *
+	 * @access private
+	 */
+	private function upgrade_to_1_2_0() {
+
+		// Options to migrate from legacy naming convention to new convention
+		$option_names = [
+			'optimizely_post_types' => 'optimizely_x_post_types',
+			'optimizely_version' => 'optimizely_x_version',
+			'optimizely_token' => 'optimizely_x_token',
+			'optimizely_project_id' => 'optimizely_x_project_id',
+			'optimizely_project_name' => 'optimizely_x_project_name',
+			'optimizely_url_targeting' => 'optimizely_x_url_targeting',
+			'optimizely_url_targeting_type' => 'optimizely_x_url_targeting_type',
+			'optimizely_variation_template' => 'optimizely_x_variation_template',
+			'optimizely_activation_mode' => 'optimizely_x_activation_mode',
+			'optimizely_conditional_activation_code' => 'optimizely_x_conditional_activation_code',
+			'optimizely_num_variations' => 'optimizely_x_num_variations',
+		];
+
+		foreach ( $option_names as $old_name => $new_name ) {
+			$option_value = get_option( $old_name );
+			if ( false !== $option_value ) {
+				update_option( $new_name, $option_value );
```

Comment:
Should we remove the old option names to remove any potential confusion in the future?

Reply:
@sboisvert I believe some organizations that test this out and may have a need to switch back to the Classic https://wordpress.org/plugins/optimizely/ version, so for now it might be best to preserve both sets.